### PR TITLE
updated with mainnet contract addresses and mainnet confgurations for Godwoken

### PR DESCRIPTION
**Pull request by Mohsinsiddi**
updated with mainnet contract addresses and mainnet confgurations for Godwoken
@nblogist review

### CODE REVIEW

File: src/components/Popup/Popup.js [hunk 1]
```diff
@@ -93,14 +93,20 @@ function Popup(props) {
                 let rawImg = value.data.image;
                 var name = value.data.name;
                 var desc = value.data.description;
+                const correct_num = Number(name.charAt(name.length - 1)) + 1;
+                const updated_name = name.replace(
+                  name.charAt(name.length - 1),
```
Comment (nblogist) [inline src/components/Popup/Popup.js:102]: Add comment here defining the problem with numbering and that you are solving it here

File: src/components/Profile/Profile.js [hunk 2]
```diff
@@ -83,30 +83,39 @@ function Profile() {
...
                 return metadata;
               });
 
               getUri.then((value) => {
                 let rawImg = value.data.image;
                 var name = value.data.name;
                 var desc = value.data.description;
+
+                const correct_num = Number(name.charAt(name.length - 1)) + 1;
+                const updated_name = name.replace(
```
Comment (nblogist) [inline src/components/Profile/Profile.js:110]: Same here

File: src/components/Swap/Swap.js [hunk 3]
```diff
@@ -153,6 +154,11 @@ function Swap() {
           let rawImg = value.data.image;
           var name = value.data.name;
           var desc = value.data.description;
+          const correct_num = Number(name.charAt(name.length - 1)) + 1;
```
Comment (nblogist) [inline src/components/Swap/Swap.js:162]: Ditto

File: src/constants/constants.js [hunk 4]
```diff
@@ -1,4 +1,4 @@
 export const POLYGON_BRIDGE_ADDRESS =
   "0xFa3b46D79e8cA79dDA9611382a15C2775823705f";
 export const GODWOKEN_NFTS_ADDRESS =
-  "0x53cd4714E2F547c980684edF545f37fa344A4A30";
+  "0x5e72E3cE0cbC0975456c1C9D4314dCB2fd0a8Ce6";
```
Comment (nblogist) [inline src/constants/constants.js:4]: Why did the address change?

Comment (nblogist) [reply src/constants/constants.js:4]: Oh wait cause they are now deployed on mainnet

**Review by nblogist — CHANGES_REQUESTED** [review]
Very good work !!

File: src/components/Popup/Popup.js (hunk 1 above)
Comment (Mohsinsiddi) [reply src/components/Popup/Popup.js:102]: updated

File: src/components/Profile/Profile.js (hunk 2 above)
Comment (Mohsinsiddi) [reply src/components/Profile/Profile.js:110]: updated

File: src/components/Swap/Swap.js (hunk 3 above)
Comment (Mohsinsiddi) [reply src/components/Swap/Swap.js:162]: updated

File: src/constants/constants.js (hunk 4 above)
Comment (Mohsinsiddi) [reply src/constants/constants.js:4]: yes!

**Review by nblogist — APPROVED** [review]

**Pull request merged**